Include/object.h

```diff
@@ -576,8 +576,8 @@ given type object has a specified feature.
 /* Objects behave like an unbound method */
 #define Py_TPFLAGS_METHOD_DESCRIPTOR (1UL << 17)
 
-/* Unused. Legacy flag */
-#define Py_TPFLAGS_VALID_VERSION_TAG  (1UL << 19)
+/* Type structure is potentially exposed (revealed) to other threads */
+#define Py_TPFLAGS_EXPOSED  (1UL << 19)
```

Comment:
Let's revert this before it's committed. I'd rather not change a publicly visible flag.

If needed, we can use a bit in `ob_flags`.

Reply:
I think the debug asserts are useful enough to allocate a ob_flags value for that purpose.  We can remove later if we decide we need more bits.